### ISSUE TITLE
Reduce / improve active zone for the tab close button

### ISSUE THREAD

**Issue by jackburton79** [Genio-The-Haiku-IDE/Genio#265]
The active zone for the tab close button is too big, as shown in the screenshot.
![screenshot](https://github.com/Genio-The-Haiku-IDE/Genio/assets/3501519/1876d8ce-25e1-46c1-b370-290822713e74)

**Comment by Freaxed** [comment]
Linked with this also we should find a way to avoid start dragging a tab when pressing the close button. Dragging with the X icon cause the tab to be dragged but then when released it closes.

**Comment by jackburton79** [comment]
I had a look at the code, and debugged a bit:
It seems there's something wrong in the logic of WebTabView::_DrawCloseButton() and  WebTabView::_CloseRectFrame(): 

The frame passed to _DrawCloseButton() is passed directly to _CloseRectFrame(), but it's not supposed to be the frame of the button.
Moreover: I printed the values of the frame and they change in a way I don't understand yet.
Needs to be reviewed

**Comment by jackburton79** [comment]
I tried to fix the problem, but I failed.
By the way, this code is taken from Webpositive which has the same problem.

I opened a ticket there: https://dev.haiku-os.org/ticket/18741

**Comment by humdingerb** [comment]
It seems like the active zone issue is fixed at least for Genio.

**Comment by jackburton79** [comment]
> It seems like the active zone issue is fixed at least for Genio.

Yes, we (@Freaxed ) completely rewrote the tab code, we have a new Tab API now.
